index.html

```diff
@@ -5,232 +5,250 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>setup and mobile first</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Desktop version project</title>
 </head>
 <body>
-    <header>
-        <a class="logo"> Dorcas</a>
-        <nav class="navlist">
-            <img src="img/Union.png" alt="navLogo">
-        </nav>
-    </header>
+    <section class="container">
+        <div class="container">
+            <header>
+                <div class="logo">
+                    <a href="#">Dorcas</a>
+                </div>
+                <nav class="Union">
+                    <ul>
+                        <li><a href="#portofolio">Portfolio</a></li>
+                        <li><a href="#about">About</a></li>
+                        <li><a href="#contact">Contact</a></li>
+                    </ul>
+                    <i class="fa fa-list-squares"></i>
+                </nav>
+            </header>
 
-    <main class="mainContent">
-        <section class="Content">
-            <h1>I'm Dorcas 
-                Glad to see you!
-            </h1>
+            <main class="content">
+                <section class="subcontent">
+                    <h1>
+                        I'm Dorcas
+                        Glad to see
+                        you!
+                    </h1>
+                    <p>
+                        I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, don’t 
+                        hesitate to contact me.
+                    </p>
```

Comment:
Kindly add the background image to your headline section and make it responsive to match the given design👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="500" alt="image" src="https://github.com/Dorcas126/Setup-and-mobile-first-project/assets/64643995/cc542dff-ac76-4427-903b-43cba0dc1ecd">|<img width="500" alt="image" src="https://github.com/Dorcas126/Setup-and-mobile-first-project/assets/64643995/f8a122c6-4692-4583-9b94-a0dbfa266810">|

also take note of the spacings, font properties and position of the elements👍.